### DISCUSSION TITLE
A big big thank you!

### DISCUSSION THREAD

**Discussion by IcemarkUK** [axmolengine/axmol#1693] [General]
Just wanted to say a big thanks to @halx99 and @rh101 for all the help with my first stage of migration to AXMOL. Helping with my understanding of the SDK and diagnosing and fixing any OSX specific issues that was having. I now have OSX and Windows (just need some work on resources for windows) fully functional for both The Lords of Midnight and Doomdark's Revenge. Now going to make sure the iOS **_projects_** work and then Android.

My next maintenance release is currently going through QA and will be released in March. After that I hope to swap over to the AXMOL powered versions from my summer release onward.

The migration was much smoother than I imagined and I'm relieved to know that I can move to a newer and better supported SDK and away from Cocos2d-x. For the record this is my second migration from an abandoned SDK for these projects in 10 years.

So many thanks.

![memoji](https://github.com/axmolengine/axmol/assets/8301416/b62f4a5f-9182-4a34-8bb6-eb65101f99e4)

**Comment by danialias** [comment]
Love to read this.

**Comment by IcemarkUK** [comment]
Just to add to this, I now have both iOS and Android up and running.
